.github/CODEOWNERS

```diff
@@ -35,7 +35,6 @@
 # (Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)
 /core/controllers/ @aks681
 /core/domain/ @DubeySandeep
-# TODO(#10619): Add @bansalnitish as owner after 2020-10-31
```

Comment:
This removal is expected? IF yes, then this PR should also close #10619, right?

Reply:
Thanks for catching that! Yes, it is expected, but I have closed that issue instead of marking this one as fixing it (because that issue is now obsolete and this PR doesn't exactly "fix" it).